redfish/eventdestination.go

```diff
@@ -12,6 +12,7 @@ import (
 	"strings"
 
 	"github.com/stmcginnis/gofish/common"
+	"github.com/stmcginnis/gofish/oem/zt"
```

Comment:
Still importing OEM handling in the standard core code. If ZT is not following the standard then an OEM class will need to be used like discussed for the `EventService`.